docs/tools/vdb_table/data/rockset.json

```diff
@@ -14,91 +14,91 @@
     "comment": ""
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "C++"
     ],
     "source_url": "",
     "comment": ""
   },
   "github_stars": 0,
-  "vector_launch_year": 0,
+  "vector_launch_year": 2023,
   "metadata_filter": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.rockset.com/documentation/docs/running-vector-search-with-rockset#querying-on-the-index",
+    "comment": "Hybrid search as easy as SQL WHERE clause: https://rockset.com/blog/vector-search-scale/"
   },
   "facets": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "geo_search": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "multi_vec": {
-    "support": "",
+    "support": "full",
```

Comment:
Please provide reference to docs/example notebook in source_url. It is a requirement for any new support claim.